.github/workflows/publish.yml

```
@@ -16,16 +16,19 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - run: git clone --single-branch --no-tags --depth 1 -b master https://grafanabot:${{ secrets.GH_BOT_ACCESS_TOKEN }}@github.com/grafana/website-sync ./.github/actions/website-sync
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn config get cacheFolder)"
     - uses: actions/cache@v2.1.6
       with:
-        path: '**/node_modules'
-        key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: yarn-${{ hashFiles('**/yarn.lock') }}
```

```suggestion
        key: yarn-${{ hashFiles('**/yarn.lock') }}
        restore-keys: |
            yarn-
```
This will allow you to reuse as much of the cache as possible when the lockfile changes